tests/test_signal.py

```diff
@@ -62,3 +62,12 @@ def test_sliding_window_removal(self):
         da = xr.DataArray(data, {"distance": s})
         da = xp.sliding_mean_removal(da, 0.1 * n * d)
         assert np.allclose(da, 0)
+
+    def test_medfilt(self):
```

Comment:
Maybe just test to see what happends if you do not provide all dims.

Reply:
You need to merge the dev branch in your working branch to get the latest updates